src/billing/components/Free/PAYGConversion.tsx

```diff
@@ -9,13 +9,26 @@ import {
   Heading,
   HeadingElement,
   FontWeight,
-  CTALinkButton,
+  Button,
 } from '@influxdata/clockface'
+import {useHistory} from 'react-router-dom'
+
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
 // Constants
 import {CLOUD_URL, CLOUD_CHECKOUT_PATH} from 'src/shared/constants'
 
 const PAYGConversion: FC = () => {
+  const history = useHistory()
+
+  const handleClick = () => {
+    if (isFlagEnabled('unityCheckout')) {
+      history.push('/checkout')
+    } else {
+      window.location.href = `${CLOUD_URL}${CLOUD_CHECKOUT_PATH}`
```

Comment:
Im sure theres a good reason for this, but why the use of window.location.href here instead of history.replace?

Reply:
so looking at the docs, I don't think we care about the history at that point since we're navigating the user to the cloud2 website, and not keeping them on the current domain, but if you think differently LMK because I've never used history.replace to go from one domain to another